commands/ssh.js

```diff
@@ -15,7 +15,9 @@ module.exports = function(topic, command) {
     topic: topic,
     command: command,
     description: 'Create an SSH session to a dyno',
-    help: `Usage: heroku ${topic}:${command}`,
+    help: `Example:
+    
+     heroku ${topic}:${command}`,
```

Comment:
Need four spaces so the generator will recognize it as code.  The devcenter documentation should insert the $ automatically because of the term-format